gobblin-restli/gobblin-flow-config-service/gobblin-flow-config-service-server/src/main/java/org/apache/gobblin/service/FlowConfigsV2Resource.java

```diff
@@ -119,6 +121,20 @@ public UpdateResponse update(ComplexResourceKey<FlowId, FlowStatusId> key, FlowC
     return this.getFlowConfigResourceHandler().updateFlowConfig(flowId, flowConfig);
   }
 
+  /**
+   * Partial update the flowConfig specified
+   * @param key composite key containing group name and flow name that identifies the flow to update
+   * @param flowConfigPatch patch describing what fields to change
+   * @return {@link UpdateResponse}
+   */
+  @Override
+  public UpdateResponse update(ComplexResourceKey<FlowId, FlowStatusId> key, PatchRequest<FlowConfig> flowConfigPatch) {
+    String flowGroup = key.getKey().getFlowGroup();
```

Comment:
Do we not want partialUpdate in V1 of flowConfig ?

Reply:
I think we can just add features to V2 from now on, there isn't really any reason to add every feature to both versions.